web-api/terraform/bin/deploy-app-color.sh

```diff
@@ -27,6 +27,8 @@ fi
 [ -z "${MIGRATE_FLAG}" ] && echo "You must have MIGRATE_FLAG set in your environment" && exit 1
 [ -z "${ZONE_NAME}" ] && echo "You must have ZONE_NAME set in your environment" && exit 1
 [ -z "${COLOR}" ] && echo "You must have COLOR set in your environment" && exit 1
+[ -z "${COMMIT_SHA}" ] && echo "You must have COMMIT_SHA set in your environment" && exit 1
```

Comment:
Could we make this script grab the commit sha on its own and set the environment variable? I want to be able to run npm run deploy:green without the need for setting more environment variables.